Calibration/HcalAlCaRecoProducers/test/AlCaHBHEMuonProducerFilter_cfg.py

```diff
@@ -13,7 +13,7 @@
 from Configuration.AlCa.autoCond import autoCond
 process.GlobalTag.globaltag=autoCond['run2_data']
 
-process.load("RecoLocalCalo.EcalRecAlgos.EcalSeverityLevelESProducer_cfi")
+#process.load("RecoLocalCalo.EcalRecAlgos.EcalSeverityLevelESProducer_cfi")
```

Comment:
Please delete this line if the tests work out like this.

Reply:
Done